### DISCUSSION TITLE
What is status : skipped ?

### DISCUSSION THREAD

**Discussion by th3-ch0s3n-0n3** [Legit-Labs/legitify#305] [Q&A]
I am getting status as skipped in "violationEntityType": "organization members", can anyone tell what's meaning of skipped here.

**Comment by royb-legit** [comment]
Might be a permission issue, did you get any error in the logs?

**Comment by th3-ch0s3n-0n3** [comment]
No I didn't received any error.

**Comment by royb-legit** [comment]
Look at the output of permissions_log.json, and search for the skipped policy - that should tell you which permission issue was encountered.
